added ABB RobotStudio and Asciidoctor

runxel opened the pull request:
**Issue:**
Closes #1870

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Manually vectorized – ABB is sparsely with any logos anywhere. Only the company logo is featured, if ever.   
Robotstudio logo is taken from its `.exe`.  
Hex value is taken of the ABB company logo itself.

runxel commented:
Sorry, I fucked up.
Just ignore the `.gitignore` and don't pull it. Again, so sorry!
Should have been two pull-requests...

ericcornelissen commented on icons/asciidoctor.svg:
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Asciidoctor icon</title><path d="M23 24a.8.8 0 01-1-.46L13.53 2.89 9.75 12h2.48a.8.8 0 01.78.82.79.79 0 01-.81.78H3a.79.79 0 01-.78-.81A.8.8 0 013 12h5L12.8.49a.8.8 0 01.72-.49.78.78 0 01.75.5l9.19 22.38A.81.81 0 0123 24zm-12.46-8.23H1.29a.81.81 0 00-.81.79.79.79 0 00.78.81h4.53L3.5 22.88a.79.79 0 00.42 1.05.81.81 0 001 ...
```
I would personally expect the logo to be [the one in the rounded square](https://github.com/asciidoctor/brand/blob/master/logo/logo-fill-bw.svg), why did you decide to omit that from the logo?

runxel replied:
Well they provide two versions, right?
`outline` and `fill`
The 'Fill' version however has two colors and therefore isn't suited for this project here. You could of course "stencil" it out. But that's just dumb. Nobody would stencil the important part out. A rounded square adds no value to a logo.

To give an analogy: The MacDonalds logo sits in a red square – However nobody would actually reproduce this logo in monochrome where the square is black and the famous "M" is transparent.

Or to link to the new Sublime Text logo: You would want to have the "S", not the square the _actual logo_ sits in. :)

ericcornelissen replied:
I agree that this is somewhat subjective. However:

- There are other examples where the logo does include a square, e.g. [IntelliJ IDEA](https://simpleicons.org/?q=intellij%20idea), [WebStorm](https://simpleicons.org/?q=webstorm), [Kentico](https://simpleicons.org/?q=kentico), and to some extend even [Adobe](https://simpleicons.org/?q=adobe).
- Asciidocter consistently uses it **with** the rounded square. Look at [the favicon of their website](https://asciidoctor.org/), their [GitHub profile](https://github.com/asciidoctor), or their [Gitter Chat](https://gitter.im/asciidoctor/asciidoctor). The only place where it is not used(/shown) is their [Twitter](https://twitter.com/asciidoctor), but I would say that is just because it is inside a circle already.
- I'm not sure what you're getting at with McDonalds. If I [search Google or MacDonalds](https://www.google.com/search?q=mcdonalds) I get the logo as Just the "M". If I got to [mcdonalds.com](https://www.mcdonalds.com/us/en-us.html) I see just the "M". I suppose it is sometimes placed in a squircle?
- Sublime is more subjective because the official website uses both versions, but it seems the version without the square is used more often.

Anyway, what do other @simple-icons/maintainers think?

phatbhoy67 replied:
Reading the [Asciidoctor Brand & Visual Identity] (https://github.com/asciidoctor/brand/blob/master/README.adoc) README on GitHub it is clear from their specifications that:

- "The logo is square."
- "The main logo is the fill variant."
- "The round edges version is preferred"


They also (helpfully) provide a version (https://github.com/asciidoctor/brand/blob/master/logo/logo-fill-bw.svg) most suitable for use within this project. Resize...optimize... 🎉 😄

runxel replied:
**tldr;** While I can't agree I will still change it.

---

Your commitment to stick to official brand guides and what not is amazing, I fully respect that. I'm just arguing in favor of the logo.
While it's certainly right, that at most places the Asciidoctor logo comes in their rounded square flavor they also fail to actually give a good comparison. At no place they are in need of a monochrome version with an unknown and potentially changing background.
The version you have linked, @phatbhoy67, is also not monochrome/stenciled. (The "A" is white.)
So if you argue: Don't alter the logo! – It doesn't really work, we alter it nonetheless.

I am a trained graphic designer and personally would never do what I am doing right now :D 
It gives the rounded square (which again, adds nothing intrinsic to a logo<sup>[0]</sup>) to much visual weight and importance – while the most important part is stenciled out.

I hope you now can at least comprehend my view and why I think this decision is bad – yet being correct.

[0] Indeed I would have done it different in many of the examples you linked :)

ericcornelissen commented on _data/simple-icons.json:
```diff
@@ -15,6 +15,11 @@
+        {
+            "title": "ABB Robotstudio",
+            "hex": "EC1B24",
+            "source": "https://new.abb.com/products/robotics/de/robotstudio/downloads"
```
Where on this page can I find the logo?

Also, it would be nice to have the English link because this is an international project.

```suggestion
            "source": "https://new.abb.com/products/robotics/en/robotstudio/downloads"
```

runxel replied:
ABB is very reluctant to show _any_ logos anywhere.
In the end the logo was taken from the `.exe`...

ericcornelissen commented on _data/simple-icons.json:
```diff
@@ -15,6 +15,11 @@
             "source": "https://about.me/assets"
         },
+        {
+            "title": "ABB Robotstudio",
```
Given the title of [the page you linked](https://new.abb.com/products/robotics/en/robotstudio/downloads) I would expect the capatalization to be as "RobotStudio" 🤔 

```suggestion
            "title": "ABB RobotStudio",
```

Remember to update it in the SVG as well 😉

ericcornelissen commented on _data/simple-icons.json:
```diff
@@ -15,6 +15,11 @@
         },
+        {
+            "title": "ABB Robotstudio",
+            "hex": "EC1B24",
```
This seems to be the brand colour for ABB itself, what makes you choose it as the brand color for the RobotStudio?

runxel replied:
We could also go with `#FF9E0F`. Maybe that's better, because it's a color from the logo.

By the way a high resolution logo can be found [at the MS Store](https://www.microsoft.com/en-us/p/robotstudio-holographic).

runxel commented:
Sorry again @ericcornelissen for all the hustle and thanks for the patience with me.
Next time it will be smoother ;)

[x] removed the .gitignore thing
[x] changed naming and color of RobotStudio

ericcornelissen commented on icons/abbrobotstudio.svg:
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ABB Robotstudio icon</title><path d="M24 12.46a9.6 9.6 0 01-19.2 0h1.07a8.53 8.53 0 108.53-8.53V2.86a9.6 9.6 0 019.6 9.6zm-9.6-3.2a3.2 3.2 0 103.2 3.2 3.2 3.2 0 00-3.2-3.2zm-2 0l-.6-6.67-2.46 1.92-1.46-1.44a4.67 4.67 0 00-5.62-.37L.24 4a.54.54 0 00-.15.74.54.54 0 00.74.15l2-1.31a3.64 3.64 0 014.29.22l1.37 1.38L6.2 7z"/> ...
```
For consistency between the data file and the SVGs we require that the titles are the same.

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ABB RobotStudio icon</title><path d="M24 12.46a9.6 9.6 0 01-19.2 0h1.07a8.53 8.53 0 108.53-8.53V2.86a9.6 9.6 0 019.6 9.6zm-9.6-3.2a3.2 3.2 0 103.2 3.2 3.2 3.2 0 00-3.2-3.2zm-2 0l-.6-6.67-2.46 1.92-1.46-1.44a4.67 4.67 0 00-5.62-.37L.24 4a.54.54 0 00-.15.74.54.54 0 00.74.15l2-1.31a3.64 3.64 0 014.29.22l1.37 1.38L6.2 7z"/></svg>
```

runxel replied:
Pretty weird. Already had done that, but I think because I did some amend/rebase vodoo to get that .gitignore out of the way I got the old thing back. Sorry again! Next PR will be cleaner ;)
Anyway, we're just a force push away!

ericcornelissen replied:
No problem, thanks for the update :+1:

ericcornelissen commented:
Thanks for your work on this @runxel :tada:
